### DISCUSSION TITLE
hope to be able to customize the data storage path.

### DISCUSSION THREAD

**Discussion by super-zhang** [TriliumNext/Trilium#7353] [Q&A] [answered]
### Describe feature

hope to be able to customize the data storage path.

**Comment by zadam** [comment, marked as answer]
Hi, you need to set an environment variable to change the data directory: https://github.com/zadam/trilium/wiki/Data-directory

**Comment by super-zhang** [comment]
Thank you. I have successfully set it up according to your method.
